kotlintest-core/src/main/kotlin/io/kotlintest/specs/TestPrefixes.kt

```diff
@@ -1,5 +1,9 @@
 package io.kotlintest.specs
 
-internal fun createTestName(prefix: String, name: String): String {
-  return if(name.startsWith("!")) "!$prefix${name.drop(1)}" else "$prefix$name"
+fun createTestName(prefix: String, name: String): String {
+  return when {
+    name.startsWith("!") -> "!$prefix${name.drop(1)}"
```

Comment:
Should there be a space here between the prefix ?
Would be good to have a test that works against FeatureSpec or something that actually uses this.

Reply:
There's no need for a space.

```
fun isFocused() = name.startsWith("f:")
fun isBang(): Boolean = name.startsWith("!")
```

Reply:
What I mean is, will it end up being "Feature:TestHere" rather than "Feature: Test here"

Reply:
Yes, but all the prefixes are like `"Feature: "`, and not `"Feature:"`. This was the original behavior before I chagned the code, so I kept it that way

Reply:
Great I was just checking.

Might still be worth a test though ;)